src/components/VDiskPopup/VDiskPopup.tsx

```diff
@@ -146,6 +149,30 @@ export const VDiskPopup = ({data, ...props}: VDiskPopupProps) => {
         [data, nodeHost, isFullData],
     );
 
+    const donorsInfo: InfoViewerItem[] = [];
+    if ('Donors' in data && data.Donors) {
+        const donors = data.Donors;
+        for (const donor of donors) {
+            const isFullDonorData = isFullVDiskData(donor);
+            donorsInfo.push({
+                label: 'VDisk',
+                value: (
+                    <InternalLink to={getVDiskLink(donor)}>
+                        {stringifyVdiskId(
+                            isFullDonorData
+                                ? donor.VDiskId
+                                : {
+                                      NodeId: donor.NodeId,
+                                      PDiskId: donor.PDiskId,
+                                      VSlotId: donor.VSlotId,
+                                  },
+                        )}
+                    </InternalLink>
+                ),
+            });
+        }
+    }
```

Comment:
This link cannot be selected.

There is issue for this, you should do it to make it work: https://github.com/ydb-platform/ydb-embedded-ui/issues/1360. Probably in another PR